CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# [0.21.0](https://github.com/awslabs/aws-cdk/compare/v0.20.0...v0.21.0) (2018-12-20)
+
+
+### Bug Fixes
+
+* **aws-cloudformation:** change the type of Role in CodePipeline Actions to IRole. ([#1364](https://github.com/awslabs/aws-cdk/issues/1364)) ([3d07e48](https://github.com/awslabs/aws-cdk/commit/3d07e48)), closes [#1361](https://github.com/awslabs/aws-cdk/issues/1361)
+* **codebuild:** Rename includeBuildID property of S3BucketBuildArtifacts ([#1354](https://github.com/awslabs/aws-cdk/issues/1354)) ([84eb7ad](https://github.com/awslabs/aws-cdk/commit/84eb7ad)), closes [#1347](https://github.com/awslabs/aws-cdk/issues/1347)
+* **toolkit:** scrutiny dialog should fail with no tty ([#1382](https://github.com/awslabs/aws-cdk/issues/1382)) ([478a714](https://github.com/awslabs/aws-cdk/commit/478a714)), closes [#1380](https://github.com/awslabs/aws-cdk/issues/1380)
+
+
+### Features
+
+* **aws-codebuild:** change the API of GitHub and BitBucket Sources. ([#1345](https://github.com/awslabs/aws-cdk/issues/1345)) ([9cebf0d](https://github.com/awslabs/aws-cdk/commit/9cebf0d))
+* add "engines.node" key to all packages ([#1358](https://github.com/awslabs/aws-cdk/issues/1358)) ([b595cf0](https://github.com/awslabs/aws-cdk/commit/b595cf0)), closes [#1337](https://github.com/awslabs/aws-cdk/issues/1337)
```

Comment:
That is more of a `chore` I think.

Reply:
You are right. I'll remove from the changelog

Reply:
You know what, I'll leave this. Since it does affect users, I think it should be indicated there